### PR TITLE
Add recommendation about documenting git workflow & environment

### PR DESCRIPTION
**Pull request by nodanaonlyzuul**
Hi Folks

I am pretty happy with a git workflow section that was added to [scsb_item_updater's README](https://github.com/NYPL-discovery/scsb_item_updater#git-workflow--deployment) and think it's a good idea if we include information like that in all our READMEs.

Any objections? Wording changes?

### CODE REVIEW

**Comment by thisisstephenbetts** [comment]
Looks good and sensible. In my experience, using `master` for the production branch is more common, but I'm fine either way.

**Comment by holingpoon** [comment]
@thisisstephenbetts I think as long as the scheme is documented in README it should be fine. 👍 @nodanaonlyzuul consider this approved!

**Comment by nodanaonlyzuul** [comment]
So odd - I've _never_ worked at a company who uses `master` as production.
It has always been a stable and technically deployable branch that holds commits for an upcoming release.

I agree with @holingpoon that it's most important to document it but maybe we can take a poll of the room and decide on stability hierarchy that we use on all projects.

**Comment by EdwinGuzman** [comment]
Looks good. We've been documenting the git branch workflow for the node/react apps for a while now and our convention is to use master as the branch that gets deployed to production. That was set up with IT for Bamboo as well.

But I agree that it should always be documented in respective repo READMEs.

File: standards/documentation.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ README documents MUST contain the following sections:
 ### Optional README sections/additions
 
 - SHOULD include current, stable version information
+- SHOULD describe git workflow, including if branches map to certain environments.  
+  As an example, see [NYPL/scsb_item_updater's 'Git Workflow & Deployment Section'](https://github.com/NYPL-discovery/scsb_item_updater#git-workflow--deployment)
+- SHOULD instructions about deployment.
```
Comment (EdwinGuzman) [inline standards/documentation.md]: I think you're missing a word here? `include`?

**Comment by nodanaonlyzuul** [comment]
Totally agree! It's a big spicy meatball of a conversation.
I don't care so much, as long as we're consistent. 

I could have been clearer about how I'm used to using `master`.
I've seen it used as nightly. Something release tags/branches would be cut off of.
So as a nightly / brancg-that-feature-branches are cut from it was important to be passing CI.

File: standards/documentation.md (hunk 1 above)
Comment (nodanaonlyzuul) [reply standards/documentation.md]: Fixed! Thanks 🦅 👁 !

**Comment by nodanaonlyzuul** [comment]
I am merging this in.
We can update this if/once we decide on a standard git workflow.
In the meantime - we should document the existing workflow per project.

**Comment by thisisstephenbetts** [comment]
Yup, that sounds very sensible. Thanks!

**Pull request merged**